DataFormats/L1TMuon/interface/EMTFHit.h

```diff
@@ -80,16 +87,17 @@ namespace l1t {
 
     CSCDetId CreateCSCDetId() const;
     RPCDetId CreateRPCDetId() const;
-    // GEMDetId CreateGEMDetId() const;
+    GEMDetId CreateGEMDetId() const;
+    ME0DetId CreateME0DetId() const;
 
     // void ImportCSCCorrelatedLCTDigi (const CSCCorrelatedLCTDigi& _digi);
     CSCCorrelatedLCTDigi CreateCSCCorrelatedLCTDigi() const;
     // void ImportRPCDigi (const RPCDigi& _digi);
     // RPCDigi CreateRPCDigi() const;
     // void ImportCPPFDigi (const CPPFDigi& _digi);
     CPPFDigi CreateCPPFDigi() const;
-    // void ImportGEMPadDigi (const GEMPadDigi& _digi);
-    // GEMPadDigi CreateGEMPadDigi() const;
+    // void ImportGEMPadDigiCluster (const GEMPadDigiCluster& _digi);  // TODO: implement placeholder when others are implemented
```

Comment:
please remove commented-out code

Reply:
Additions were made to reflect structure of current code, and placeholder comments that were already present were added for the new portions